pkg/cmd/server/api/types.go

```diff
@@ -112,12 +112,30 @@ type NodeConfig struct {
 	// MasterClientConnectionOverrides provides overrides to the client connection used to connect to the master.
 	MasterClientConnectionOverrides *ClientConnectionOverrides
 
-	// DNSDomain holds the domain suffix
+	// DNSDomain holds the domain suffix that will be used for the DNS search path inside each container. Defaults to
+	// 'cluster.local'.
 	DNSDomain string
 
-	// DNSIP holds the IP
+	// DNSIP is the IP address that the DNS server will listen on and which will be reported into pods. Defaults
```

Comment:
"DNSIP is the IP address that the DNS server will listen on" isn't accurate, right?